Source/quests.cpp

```diff
@@ -150,7 +150,7 @@ void DrawWarLord(int x, int y)
 	}
 }
 
-void DrawSChamber(int q, int x, int y)
+void DrawSChamber(quest_id q, int x, int y)

```

Comment:
Another opportunity for QuestStruct& (even with the known single use of Q_SCHAMB).

Reply:
Oh guess I missed this on during my second round.
Oh well still left some stuff so will probably still be handled in the next cleanup